code/modules/reagents/chemistry_reagents/other.dm

```diff
@@ -225,7 +225,7 @@
 	. = ..()
 	if(!.)
 		return
-	if(alien == IS_MOTH)
+	if(has_species(M, "Moth"))
 		if(prob(7))
```

Comment:
I kind of forgot to tell you to also put `alien` inside the proc vars parenthesis, after `mob/living/M`, that's why It probably errored. Take a look at the reagent below for example.

Reply:
alien has no type so i'm not sure what the fuck it does

Reply:
It's used to check the species' reagent_tag, if they have any.